common/CI/package_checks.py

```diff
@@ -417,6 +420,19 @@ def _includes_homepage(self, file: str) -> bool:
             return 'homepage' in yaml.load(f)
 
 
+class Monitoring(PullRequestCheck):
+    _error = '`monitoring.yml` is missing'
+    _level = Level.ERROR
```

Comment:
I've set this at `ERROR` to make it mandatory for updates, but I have no issue with downgrading this to a warning. Note that changes to the help center are also needed when making this mandatory.

Reply:
Made it a warning